src/app/dataset-view/additional-components/overview-component/overview.component.ts

```diff
@@ -104,7 +106,7 @@ export class OverviewComponent extends BaseComponent implements OnInit {
 
     public get canEditDatasetInfo(): boolean {
         if (this.hasDatasetInfo) {
-            return this.datasetPermissions.permissions.canCommit;
+            return this.datasetPermissions.permissions.metadata.canCommit;
```

Comment:
These "canXXX" methods are also becoming a mess

Reply:
Discussed this approach